code/workspaces/client-api/src/infrastructure/storageService.js

```diff
@@ -0,0 +1,49 @@
+import axios from 'axios/index';
```

Comment:
Out of interest, is this any different to `import axios from 'axios';`?

Reply:
Ha, no, I don't think it is, I think that is WebStorm being helpful when I copied in axios code from somewhere else, I'll remove the `/index`